include/phasar/PhasarLLVM/Utils/LLVMIRToSrc.h

```diff
@@ -39,6 +39,11 @@ class DILocation;
 
 namespace psr {
 
+/// \file This file contains useful structs and functions to get and store
+/// information about the source code or the intermediate representation of a
+/// project.
```

Comment:
"... intermediate representation of the **target being analysed**" reads better.

This change is also relevant in other places where "project" is used in the similar context.